vnext/Shared/JSI/NapiJsiRuntime.cpp

```diff
@@ -0,0 +1,1420 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//#define NAPI_EXPERIMENTAL //TODO: Check if needed.
```

Comment:
I would expect that code does not compile without it. I think we use some experimental functions.

Reply:
It does compile, but I'll restore the define since you say it IS used.